### PR TITLE
Added uninstall-rsh-server module

### PR DESCRIPTION
**Pull request by olehermanse**

### CODE REVIEW

File: security/uninstall-rsh-server.cf/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+```
+$ sudo apt install rsh-server
+$ cf-agent -KI
+    info: Successfully removed package 'rsh-server'
+```
+
+## Adding exceptions
+
+If this package is really needed on some hosts, you can add an exception with the `exception_uninstall_rsh_server` class.
+This class can be set from the **Host specific data** section in host info pages inside Mission Portal, the CFEngine Enterprise Web UI.
```
Comment (nickanderson) [inline security/uninstall-rsh-server.cf/README.md:23]: What about switching to a string variable where you could type something more readable and more easily reference other things (like we chatted about)?

File: security/uninstall-rsh-server.cf/uninstall_rsh_server.cf [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+# Based on the OpenSCAP Security Guide for RHEL 7:
+# https://static.open-scap.org/ssg-guides/ssg-rhel7-guide-anssi_nt28_minimal.html
+# CFEngine policy based on this module:
+# https://build.cfengine.com/modules/uninstall-ftp/
+{
+  vars:
+    redhat|suse::
+      "pkg_name" slist => { "rsh-server" }; # Name in yum
+    debian::
+      "pkg_name" slist => { "rsh-server" }; # name in apt
```
[the remark this answers is not in the document]
Comment (olehermanse) [reply security/uninstall-rsh-server.cf/uninstall_rsh_server.cf:12]: So I can copy paste it, and/or update the lists with other pacakge names based on package manager.

File: security/uninstall-rsh-server.cf/README.md (hunk 1 above)
Comment (olehermanse) [reply security/uninstall-rsh-server.cf/README.md:23]: Thinking about it, can be added later.

File: security/uninstall-rsh-server.cf/uninstall_rsh_server.cf (hunk 2 above)
[the remark this answers is not in the document]
Comment (nickanderson) [reply security/uninstall-rsh-server.cf/uninstall_rsh_server.cf:12]: That's not a great reason imho.

**Pull request merged**